package.json

```diff
@@ -5,6 +5,11 @@
   "main": "src/index.ts",
   "author": "Agriculture Capstone Team",
   "license": "UNLICENSED",
+  "engines": {
+    "node": "= 8.9.1",
+    "npm": "= 5.5.1",
+    "yarn": "= 1.3.2"
```

Comment:
When we started development these were already specified in the agriboard. For the sake of not breaking peoples development environments I think the scope of this issue should be to sync the package.json engines to the ones that were originally defined at the beginning of the project
```
{
    "node": ">= 8.9.0",
    "npm": ">= 5.5.1",
    "yarn": ">= 1.3.0"
}
```